scrud_django/tests/test_rest.py

```diff
@@ -0,0 +1,124 @@
+import json
+import os
+from unittest import skip
+
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
```

Comment:
Would you be willing to try out [safetydance-django](https://github.com/quansight/safetydance_django/)?

Reply:
sounds good .. I will take a look how to use it :) thanks for the recommendation.